content/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+**What**: Explain what you changed and provide a list of changed page names.
```

Comment:
Are PR owners expected expected to add context under these fields? If so, maybe we should add comments to signal?

We should use the [tutorials](https://github.com/hashicorp/tutorials/blob/main/.github/pull_request_template.md) one as a sample

Reply:
What do you mean by context underneath? The text next to "What" should be replaced by the explanation. Do you think that will confuse folks?  I want to let the contributor actually type in what they did and why. The learn one doesn't allow for that at all it looks like.

Reply:
Yeah! I think you should indicate what needs to be replaced. This one doesn't really show that imo

Reply:
okay gotcha! I'll do italics and think about how to indicate that better!!